prototypes/pagespace-quick-create/src/styles/global.css

```
@@ -0,0 +1,62 @@
+
+*, *::before, *::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: var(--background);
+  color: var(--foreground);
+  /* Geist Sans isn't available as a web font; Inter is visually identical */
+  font-family: "Inter", ui-sans-serif, system-ui, -apple-system, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)